tests/manage/test_pvc_delete_verify_size_is_returned_to_backendpool.py

```diff
@@ -0,0 +1,223 @@
+from utility.utils import run_cmd
+from utility import templating
+from ocs.utils import create_oc_resource
+from ocs import pod, ocp
+
+log = logging.getLogger(__name__)
+_templating = templating.Templating()
+
+# Project name
+PROJECT_NAME = "ocs-372"
+
+PRJ = ocp.OCP(kind='Project')
+
+# yaml path
+TEMPLATES_DIR = "/tmp/"
+
+template_data = {}
+
+# Service
+OCP = ocp.OCP(
+    kind='Service', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Cephblockpool
+CBP = ocp.OCP(
+    kind='CephBlockPool', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Storage class
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+# Secret
+SECRET = ocp.OCP(
+    kind='Secret', namespace="default"
+)
+
+# PVC
+PVC = ocp.OCP(
+    kind='PersistentVolumeClaim', namespace=PROJECT_NAME
+)
+
+# Pod
+POD = ocp.OCP(
+    kind='Pod', namespace=PROJECT_NAME
+)
+
+
+def create_cephblock_pool(pool_name):
+
+    template_data['rbd_pool'] = pool_name
+    create_oc_resource(
+        'cephblockpool.yaml', TEMPLATES_DIR, _templating, template_data
+    )
+
+    # Check CephBlockPool is created
+    name = get_ceph_tools_pod()
+    po = pod.Pod(name, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    cmd = "ceph osd pool ls"
+
+    out, _, _ = po.exec_command(cmd=cmd, timeout=20)
+    assert pool_name in out
+
+
+def create_storageclass(sc_name, pool_name):
+
+    template_data['rbd_storageclass_name'] = sc_name
+    template_data['rbd_pool'] = pool_name
+    create_oc_resource(
+        'storageclass.yaml', TEMPLATES_DIR, _templating, template_data
+    )
+
+    # Validate storage class created
+    assert SC.get(f'{sc_name}')
+
+
+def create_secret():
+
+    name = get_ceph_tools_pod()
+    po = pod.Pod(name, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+
+    # Key value corresponds to a admin defined in Ceph cluster
+    cmd = "ceph auth get-key client.admin|base64"
+    out, _, _ = po.exec_command(cmd=cmd, timeout=20)
+    template_data['base64_encoded_admin_password'] = out
+
+    # ToDo: Uncomment these steps when we are creating client.kubernetes
+    # # Key value corresponds to a user name defined in Ceph cluster
+    # cmd = (
+    #     f'ceph auth get-or-create-key client.kubernetes
+    #     mon "allow profile rbd" osd "profile rbd pool={pool_name}"'
+    # )
+    # _, _, _ = po.exec_command(cmd=cmd, timeout=20)
+    # cmd = "ceph auth get-key client.kubernetes|base64"
+    # template_data['base64_encoded_user_password'] = out
+
+    create_oc_resource(
+        'secret.yaml', TEMPLATES_DIR, _templating, template_data
+    )
+
+
+def check_ceph_available_space():
+
+    name = get_ceph_tools_pod()
+    po = pod.Pod(name, namespace=defaults.ROOK_CLUSTER_NAMESPACE)
+    cmd = "ceph status --format json"
+    out, err, ret = po.exec_command(cmd=cmd, timeout=20)
+    if not ret:
+        pods = json.loads(out)
+        used = pods['pgmap']['bytes_used']
+        used_in_gb = used / (1024 * 1024 * 1024)
```

Comment:
Please create a constant for GB.
At the top of this file add:
GB = (1024 * 1024 * 1024)

Reply:
You can also do 1024 ** 3

Reply:
done